helm-charts/splunk-otel-collector/values.yaml

```diff
@@ -371,7 +371,9 @@ logsCollection:
     #     containerName:
     #       value: server
     #     firstEntryRegex: ^[^\s].*
-
+    # Set `splunk.com/include` annotation to true on pod and `logsCollection.containers.useSplunkIncludeAnnotation` flag to `true` to include its logs from ingested to your Splunk platform deployment.
+    # All other logs will be ignored.
+    useSplunkIncludeAnnotation: false
```

Comment:
Looks good to me. Just please add a line breaks above and below this section to avoid mixing with other comments